modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -30,6 +30,10 @@ import fs2.io.file.Path
 
 object method {
 
+  val snapshotInterval: NonNegLong = 1000L
+
+  val delayToUpdateCurrent: NonNegLong = 500L
```

Comment:
this should be a part of `AppConfig`

Reply:
Before moving it to `AppConfig`, I will wait to see if they need to be moved to the `sdk` module.